app/controllers/licensees_imports_controller.rb

```diff
@@ -0,0 +1,27 @@
+class LicenseesImportsController < ApplicationController
+  
+  before_action :authenticate_user!
+  before_action :authorize_admin
+  
+  def new
+    @licensees_import = LicenseesImport.new
+  end
+
+  def create
+    @licensees_import = LicenseesImport.new(licensees_import_params)
+    #@licensees_import = LicenseesImport.new(params[:licensees_import])
+    #byebug
+    if @licensees_import.save
+      redirect_to licensees_path
+    else
+      render :new
```

Comment:
Consider adding a flash message here (and in the view) to let the user know what went wrong.

https://www.railstutorial.org/book/sign_up#sec-the_flash